test/generators/typescript/preset/ExamplePreset.spec.ts

```diff
@@ -0,0 +1,40 @@
+import { TypeScriptGenerator, TS_COMMON_PRESET } from '../../../../src/generators';
+const doc = {
+  $id: 'Test',
+  type: 'object',
+  additionalProperties: true,
+  required: ['string prop'],
```

Comment:
What about required properties? Should be rendered only these props, or another also, non-required?

Reply:
All properties are rendered if it can find an example value. It does not care whether it's required or optional as it is just a "dummy" generator 😄

Reply:
We can add support for this (as option) if someone will raise that issue :)

Reply:
Yea exactly 👍